apis/v1alpha1/instrumentation_types.go

```diff
@@ -145,6 +149,32 @@ type DotNet struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
 
+type ApacheHttpd struct {
+	// Image is a container image with Apache SDK and auto-instrumentation.
+	// +optional
+	Image string `json:"image,omitempty"`
+
+	// Env defines Apache HTTPD specific env vars. There are four layers for env vars' definitions and
+	// the precedence order is: `original container env vars` > `language specific env vars` > `common env vars` > `instrument spec configs' vars`.
+	// If the former var had been defined, then the other vars would be ignored.
+	// +optional
+	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Attrs defines Apache HTTPD agent specific attributes. The precedence is:
+	// `agent default attributes` > `instrument spec attributes`
+	// +optional
+	Attrs []corev1.EnvVar `json:"attrs,omitempty"`
```

Comment:
what exactly is this configuring?

Reply:
The OTel module for Apache httpd has number of configurable attributes (listed at https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/otel-webserver-module in Configuration section). There are reasonable defaults and some will be set by the instrumentation process, this gives a means to override those defaults.

Reply:
Should we add this link to the godoc?

Reply:
Yes, good idea.

Reply:
Done